consul/dnsdisc.py

```diff
@@ -38,12 +39,14 @@ def parse_opts():
                       help='Consul port.')
     parser.add_option('-T', '--consul-token', default=os.environ.get('CONSUL_HTTP_TOKEN'),
                       help='Consul API token.')
-    parser.add_option('-n', '--query-service', type='string', action="append", default=[],
-                      help='Name of Consul service to query.')
+    parser.add_option('-n', '--query-service-name', type='string',
+                      help='Name of Consul service name to query.')
```

Comment:
Why are you changing it from a list?

Reply:
Hmmm, yeah, the need for a list is quite an edge case currently. I think it's fine as it is right now.

If we need multiple services it would probably be multiple service IDs only, but we can add that when we need it.